unittests/tools/test_dockerbench_parser.py

```diff
@@ -0,0 +1,30 @@
+from ..dojo_test_case import DojoTestCase, get_unit_tests_path
+from dojo.tools.dockerbench.parser import DockerBenchParser
+from dojo.models import Test
+
+
+class TestDockerBenchParser(DojoTestCase):
+
+    def test_parse_file_with_no_vuln_has_no_findings(self):
+        testfile = open(
+            get_unit_tests_path() + "/scans/dockerbench/docker-bench-report-zero-vulns.json"
+        )
+        parser = DockerBenchParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(0, len(findings))
+
+    def test_parse_file_with_one_vuln_has_one_finding(self):
+        testfile = open(
+            get_unit_tests_path() + "/scans/dockerbench/docker-bench-report-single-vuln.json"
+        )
+        parser = DockerBenchParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(1, len(findings))
+
+    def test_parse_file_with_multiple_vuln_has_multiple_findings(self):
+        testfile = open(
+            get_unit_tests_path() + "/scans/dockerbench/docker-bench-report-many-vulns.json"
+        )
+        parser = DockerBenchParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertTrue(len(findings) == 50)
```

Comment:
Please add more check on the findings

Reply:
@damiencarol , thank you,  added some extra assertions in this test

Reply:
@damiencarol , fixed failed unit tests.